main.go

```diff
@@ -121,7 +128,11 @@ func configureEnvironment() (dataSource, port string) {
 			dataSource = v
 		}
 
-		return dataSource, port
+		if v := os.Getenv("MIGRATION_SOURCE"); len(v) > 0 {
```

Comment:
is there a use case for migration source being configfurable? if's there's any db changes they should be in this repo and not come from anywhere else

Reply:
There is a case for this being configurable -  docker, cloud foundry etc will have the files in different places (main is installed in /usr/local/bin in docker for example).  I had trouble getting relative paths to work properly.  Now if there was a way to somehow package the files with the executable ...